### PR TITLE
Bugfix FXIOS-7553 [v120] Fix console warnings for layout constraints in CollapsibleCardView

### PR DESCRIPTION
**Pull request by imsanchit**
## :scroll: Tickets
[Jira ticket](https://mozilla-hub.atlassian.net/browse/FXIOS-7553)
[Github issue](https://github.com/mozilla-mobile/firefox-ios/issues/16798)

## :pencil: Checklist
You have to check all boxes before merging
- [x] Filled in the above information (tickets numbers and description of your work)
- [x] Updated the PR name to follow our [PR naming guidelines](https://github.com/mozilla-mobile/firefox-ios/wiki/Pull-Request-Naming-Guide)
- [ ] Wrote unit tests and/or ensured the tests suite is passing
- [ ] When working on UI, I checked and implemented accessibility (minimum Dynamic Text and VoiceOver)
- [ ] If needed I updated documentation / comments for complex code and public methods

### CODE REVIEW

**Review by adudenamedruby — APPROVED** [review]

**Comment by imsanchit** [comment]
Hey @razvanlitianu , thanks once again for pointing out the issue, the alignment issue is fixed now, and the remaining console warnings in `CollapsibleCardView` are fixed too. Please have a look.

File: Client/Frontend/Fakespot/Views/FakespotReviewQualityCardView.swift [hunk 1]
```diff
@@ -260,7 +260,6 @@ final class FakespotReviewQualityCardView: UIView, Notifiable, ThemeApplicable {
         }
 
         setNeedsLayout()
```
[the remark this answers is not in the document]
Comment (imsanchit) [reply Client/Frontend/Fakespot/Views/FakespotReviewQualityCardView.swift]: Sure, done the same in the latest commit.

**Review by razvanlitianu — APPROVED** [review]

File: BrowserKit/Sources/ComponentLibrary/CollapsibleCardView.swift [hunk 2]
```diff
@@ -93,7 +94,14 @@ public class CollapsibleCardView: ShadowCardView, UIGestureRecognizerDelegate {
...
+        stackView.axis = .vertical
+        stackView.spacing = UX.verticalPadding
+        stackView.alignment = .center
+        stackView.isLayoutMarginsRelativeArrangement = true
+        stackView.layoutMargins = UX.margins
+    }
+    
     private lazy var headerView: UIView = .build { _ in }
     private lazy var containerView: UIView = .build { _ in }
     private var containerHeightConstraint: NSLayoutConstraint?
```
[the remark this answers is not in the document]
Comment (imsanchit) [reply BrowserKit/Sources/ComponentLibrary/CollapsibleCardView.swift]: Thanks @thatswinnie for pointing this out, fixed these warnings. Please have a look.

**Comment by thatswinnie** [comment]
Thanks for the updates @imsanchit. Unfortunately there is still the Swiftlint warning on line 104 of `CollapsibleCardView.swift` (Trailing Whitespace Violation). Also could you please rebase your branch with main so that build errors happening on Bitrise are fixed. Thank you! 🙏

**Comment by imsanchit** [comment]
@thatswinnie thanks for pointing this out. my bad I hadn't setted-up swiftlint and these warning should have been at first place itself. Have done it now, and should be good to go. Also, rebase with the `main` branch. Please have a look at the updated PR.

**Comment by thatswinnie** [comment]
@imsanchit thanks for making the changes and helping us improve Firefox 🙏

**Pull request merged**